packages/paste-website/stories/PageTemplates.stories.tsx

```diff
@@ -860,3 +880,507 @@ export const SettingsOverviewExample = (): JSX.Element => {
+            <FormControl>
+              <Label htmlFor={input4}>Label</Label>
+              <Input type="text" id={input4} name="input4" />
+            </FormControl>
+          </Form>
+        </Box>
+        <Box display="flex" justifyContent="end">
+          <ButtonGroup>
+            <Button variant="secondary">Cancel or Close</Button>
+            <Button variant="primary">
+              Next <ArrowForwardIcon decorative />
+            </Button>
+          </ButtonGroup>
+        </Box>
+      </Box>
+    </Box>
+  );
+};
+DefaultWizardExample.parameters = {
+  padding: false,
+};
+
+export const DefaultWizardWithContentExample = (): JSX.Element => {
+  const input1 = useUID();
+  const sidebarId = useUID();
+  const topbarSkipLinkID = useUID();
+  const sidebarNavigationSkipLinkID = useUID();
+  const mainContentSkipLinkID = useUID();
+  return (
+    <>
+      <Sidebar
+        variant="compact"
+        collapsed={true}
+        sidebarNavigationSkipLinkID={sidebarNavigationSkipLinkID}
+        topbarSkipLinkID={topbarSkipLinkID}
+        mainContentSkipLinkID={mainContentSkipLinkID}
+        aria-label={sidebarId}
+      >
+        <div id={sidebarNavigationSkipLinkID} />
+      </Sidebar>
+      <SidebarPushContentWrapper collapsed={true} variant="compact">
+        <Topbar id={topbarSkipLinkID}> </Topbar>
+        <Box display="flex" width="100%" justifyContent="center" paddingX="space50" id={mainContentSkipLinkID}>
+          <Box
+            paddingTop="space130"
+            paddingBottom="space160"
+            display="flex"
+            flexDirection="column"
+            rowGap="space130"
+            width="size70"
+            maxWidth="size80"
+          >
+            <Box display="flex" flexDirection="column">
+              <DetailText marginTop="space0">Add Reverse ETL destination</DetailText>
+              <Heading as="h4" variant="heading10" marginBottom="space0">
+                Set up destination
+              </Heading>
+            </Box>
+            <Box>
+              <Form>
+                <FormControl>
+                  <Label htmlFor={input1}>Destination name</Label>
+                  <Input type="text" id={input1} name="input1" />
+                  <HelpText>
+                    Identifies this destination within your organization and typically includes the business unit,
+                    geographic region, or environment. Example: Google Analytics North America Prod.
+                  </HelpText>
+                </FormControl>
+              </Form>
+            </Box>
+            <Box display="flex" justifyContent="space-between">
+              <Button variant="secondary">
+                <ArrowBackIcon decorative />
+                Back
+              </Button>
+              <Box display="flex" columnGap="space40">
+                <Button variant="secondary">Cancel or Close</Button>
+                <Button variant="primary">Create destination</Button>
+              </Box>
```

Comment:
chore: Use ButtonGroup for the other Wizard examples, too

Reply:
oops